util/execdetails/execdetails.go

```diff
@@ -180,11 +186,14 @@ func (d ExecDetails) String() string {
 // ToZapFields wraps the ExecDetails as zap.Fields.
 func (d ExecDetails) ToZapFields() (fields []zap.Field) {
 	fields = make([]zap.Field, 0, 16)
+	if d.CopTime > 0 {
+		fields = append(fields, zap.String(strings.ToLower(CopTimeStr), strconv.FormatFloat(d.CopTime.Seconds(), 'f', -1, 64)+"s"))
+	}
 	if d.ProcessTime > 0 {
 		fields = append(fields, zap.String(strings.ToLower(ProcessTimeStr), strconv.FormatFloat(d.ProcessTime.Seconds(), 'f', -1, 64)+"s"))
 	}
 	if d.WaitTime > 0 {
-		fields = append(fields, zap.String(strings.ToLower(WaitTimeStr), strconv.FormatFloat(d.ProcessTime.Seconds(), 'f', -1, 64)+"s"))
+		fields = append(fields, zap.String(strings.ToLower(WaitTimeStr), strconv.FormatFloat(d.WaitTime.Seconds(), 'f', -1, 64)+"s"))
```

Comment:
Should it be fixed in an individual PR and cherry-pick it?

Reply:
ok. I will fill another PR.

Reply:
Fix in https://github.com/pingcap/tidb/pull/16899